eng/tools/typespec-validation/src/index.ts

```diff
@@ -18,8 +18,10 @@ export async function main() {
     },
   };
   const parsedArgs = parseArgs({ args, options, allowPositionals: true } as ParseArgsConfig);
-  const folder = parsedArgs.positionals[0].split(path.sep).join("/");
-  console.log("Running TypeSpecValidation on folder:", folder);
+  const folder = parsedArgs.positionals[0];
+  const absolutePath = path.resolve(folder).split(path.sep).join("/");
```

Comment:
Extract a helper function named something like `normalizePath([...paths])` that can be unit tested, and add unit tests.  Unit tests should not rely on current directory.